test/fixtures/controllers/PromiseMiddlewareTestController.js

```diff
@@ -28,7 +38,7 @@ module.exports = function (BaseController, Promise) {
     }
 
     getFormatAsync(req, res) {
-      res.formatAsync(Promise.resolve('formatAsync success'));
+      res.formatAsync('innerHTML', Promise.resolve('formatAsync success'));
```

Comment:
Not sure why this is needed.

Reply:
`formatAsync` expects 2 arguments, this fixes a bug.